### PR TITLE
Added province data for Thailand

### PR DESCRIPTION
**Pull request by poonlap**
I created the module which simple added province names in English and Thai to Fed. State table. I tested with my environment (Odoo 11) and it works fine.

### CODE REVIEW

**Comment by kittiu** [comment]
@poonlap thank you for your contribution.
Before contribute, please sign the CLA.

**Comment by pedrobaeza** [comment]
Check also if https://github.com/OCA/partner-contact/tree/11.0/base_location_geonames_import may serve to you.

**Comment by kittiu** [comment]
> Check also if https://github.com/OCA/partner-contact/tree/11.0/base_location_geonames_import may serve to you.

@pedrobaeza this is really nice feature and repo. Thank you for pointing out.

**Comment by kittiu** [comment]
@pedrobaeza at first, I did this module https://github.com/OCA/l10n-thailand/blob/12.0/l10n_th_partner/models/res_partner.py
just to be place for more field specific to Thailand. Now as you point out, should I instead add this "Branch" field as **partner_branch** in **partner-contact** ?

**Note:** Branch (I.e., HQ, Branch 1, Branche 2) should be quite normal, but I couldn't find it in partner-contact though.

**Comment by pedrobaeza** [comment]
What do you mean by "branch"? Any definition for that?

**Comment by kittiu** [comment]
> What do you mean by "branch"? Any definition for that?

It is company Tax Branch ID. For example, one company may have 1+ branch registered. First branch is Head Quarter (branch = 00000, all company will have at least 1 branch) and 2nd branch is branch = 00001 and so on.

Each branch may submit tax separately. And so in every tax invoice document will need to have Branch ID along with Company Name. (In fact, different branch is different partner in Odoo).

Simply put, it is just another field for a partner.

**Comment by pedrobaeza** [comment]
Well, this is totally unrelated to this PR, but we do that with normal Odoo with different partners and use `partner_relation` for setting this relationship and others.
